### PR TITLE
Build: Remove cypress from monorepo

### PR DESCRIPTION
**Pull request by ndelangen**
Delete the usage of cypress in our CI and contributing docs.

We've changed from using examples to using generated sandboxes.
These sandboxes are tested using playwright instead.

### CODE REVIEW

File: scripts/run-e2e.ts [hunk 1]
```diff
@@ -196,8 +181,6 @@ const runTests = async ({ name, ...rest }: Parameters) => {
   try {
     if (shouldUseTestRunner) {
```
Comment (shilman) [inline scripts/run-e2e.ts:182]: Shouldn't this always be `true` now?

Comment (shilman) [reply scripts/run-e2e.ts:182]: Looks like with this change it will always be `false` and the test runner will never run

**Review by shilman — CHANGES_REQUESTED** [review]

File: scripts/run-e2e.ts (hunk 1 above)
Comment (ndelangen) [reply scripts/run-e2e.ts:182]: @shilman please help me understand what did I change that impacts the testrunner?

this value is coming from a cli-flag?
https://github.com/storybookjs/storybook/blob/497837346f01e320c3e7cb6d42d9f2e56b0f7101/scripts/run-e2e.ts#L61

Comment (shilman) [reply scripts/run-e2e.ts:182]: @ndelangen You removed the flag https://github.com/storybookjs/storybook/pull/19303/files#diff-dba89a5f00386a7c76cb45dcd9dbc8ec6c9027b20c0b371435fc867349bbf840L44

Comment (ndelangen) [reply scripts/run-e2e.ts:183]: Oops that was not intentional!

Comment (ndelangen) [reply scripts/run-e2e.ts:183]: Should remove the if statement instead (and remove the cli flag).

**Review by JReinhold — APPROVED** [review]

**Pull request merged**